packages/picasso-forms/src/Form/test.tsx

```diff
@@ -1,27 +1,97 @@
 import React from 'react'
-import { render } from '@toptal/picasso/test-utils'
+import { fireEvent, render } from '@toptal/picasso/test-utils'
 import { OmitInternalProps } from '@toptal/picasso-shared'
 import { Button } from '@toptal/picasso'
 
 import Form, { Props } from './Form'
+import { scrollTo } from '../utils/scroll-to'
 
-const renderForm = (props: OmitInternalProps<Props>) => {
-  const { onSubmit } = props
+jest.mock('../utils', () => ({
+  ...jest.requireActual('../utils'),
+  createScrollToErrorDecorator: jest.requireActual(
+    '../utils/scroll-to-error-decorator'
+  ).default
+}))
+jest.mock('../utils/scroll-to', () => ({
+  scrollTo: jest.fn()
+}))
+
+const renderForm = (
+  props: OmitInternalProps<Props> & { mandatory?: boolean }
+) => {
+  const { onSubmit, disableScrollOnError, mandatory } = props
 
   return render(
-    <Form onSubmit={onSubmit}>
-      <Form.Input name='test' placeholder='test input' />
+    <Form
+      data-testid='form'
+      onSubmit={onSubmit}
+      disableScrollOnError={disableScrollOnError}
+    >
+      <Form.Input name='test' placeholder='test input' required={mandatory} />
       <Button type='submit'>Submit</Button>
     </Form>
   )
 }
 
+/**
+ * form submit is not linked to any process,
```

Comment:
Not sure what does this means: linked to any process?
```suggestion
 * form submit does not return Promise or accept callback,
```

Reply:
✅